components/colorarea/dist/metadata.json

```diff
@@ -9,8 +9,7 @@
     ".spectrum-ColorArea.is-disabled",
     ".spectrum-ColorArea.is-disabled .spectrum-ColorArea-gradient",
     ".spectrum-ColorArea.is-focused",
-    ".spectrum-ColorHandle-color",
-    "[dir=\"rtl\"] .spectrum-ColorArea-handle"
+    ".spectrum-ColorHandle-color"
```

Comment:
So in SWC, we would likely to have a breaking change here on the css?
```css
.handle:dir(rtl),
:host([dir="rtl"]) .handle {
    inset-inline-end: 0;
}
```

Reply:
In your example, the code may change but the resulting styles would not:
```
.handle:dir(rtl) {
    inset-inline-end: 0;
}
```

This is sufficient to apply the styles ^